src/vcpkg/registries.cpp

```diff
@@ -96,11 +100,48 @@ namespace
             });
         }
 
+        struct VTPResult
```

Comment:
```suggestion
        struct VersionsTreePathResult
```
VTPResult is a really confusing name to me.